apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts

```diff
@@ -16,6 +16,8 @@ export class AmountFieldComponent implements OnInit, OnChanges {
     @Input() public currencyCode: string;
     /* true, ratePrecision value */
     @Input() public ratePrecision: boolean = true;
+    /** this will store giddhCurrency pipe value */
```

Comment:
Please check if it's used

Reply:
yes it is used

Reply:
Yes @prateek-giddh confirmed, it is used on the subscription page